### ISSUE TITLE
CKEditor misses focus by tabbing through content elements

### ISSUE THREAD

**Issue by mgoldbeck** [neos/neos-ui#1598]
### Description

When tabbing through the content elements of a Document they are not focused and the CKEditor toolbar is not shown.

### Steps to Reproduce

1. Select a contentelement
2. start tabbing
3. tab backwards

#### Expected behavior

By tabbing selected content elements should be focused and a the CKEditor toolbar should be shown

#### Actual behavior

![ckeditor_tabbing_focus](https://user-images.githubusercontent.com/1336044/35637547-cdd9f266-06b4-11e8-971b-facfd7c842fe.gif)

### Affected Versions

Neos: master

UI: 1.0.4
